dotnet-test-samples/sqs-lambda/src/SQSEventHandler/Adapters/SqsEventAdapter.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections.Generic;
+using Amazon.Lambda.SQSEvents;
+
+namespace SqsEventHandler.Adapters;
+
+public class SqsEventAdapter : ISqsEventAdapter
```

Comment:
Do you think there's a risk adding these layers of abstraction will make the code harder to understand?

Reply:
Simplified the code, kindly review